### PR TITLE
Added initial .gitignore

### PR DESCRIPTION
**Pull request by nanangizz**
Please try put the file on your PJSIP root dir on your environment, feel free to add more ignore patterns :)

### CODE REVIEW

**Comment by sauwming** [comment]
gak bisa di-review?

okay ignore file-nya, tapi koq di tempatku gitignore-nya sendiri gak
di-ignore ya? :D
"Untracked files:

  (use "git add <file>..." to include in what will be committed)


.gitignore

"

**Comment by nanangizz** [comment]
> why can't I review this?

check commit 90d50c298c67d2e3d9f8f574950c94c91fa85b3e & write any note there?

> the ignore file seems okay, but why the gitignore itself is not ignored? 😆 "Untracked files: (use "git add <file>..." to include in what will be committed) .gitignore "

haha.. it will be 'ignored' (tracked) once pushed/merged.

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ pjlib/include/pj/config_site.h
 lib/
 bin/
 output/
+Debug/
```
Comment (nanangizz) [inline .gitignore]: what about "Release/" or any other build configs?
Or even better if Debug, Release, etc can be set to be under "output".

Comment (trengginas) [reply .gitignore]: It's actually used for "pjsip-apps\build\Debug" folder which contains some swig_java_pjsua2 output files. Need to change the project properties (swig_java_pjsua2) apparently

**Pull request merged**